components/Blobs/Blob1.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <div class="blob1">
+        <svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
+            <path fill="#F59E0B"
+                d="M58.9,-38.8C74.4,-27.5,83.7,-3.1,80,20.4C76.4,43.9,59.7,66.6,39.4,73.1C19.1,79.5,-4.8,69.6,-24.8,57.6C-44.8,45.7,-60.8,31.6,-63.9,15.3C-67,-0.9,-57.1,-19.3,-44.3,-30C-31.5,-40.7,-15.8,-43.7,3,-46.1C21.7,-48.4,43.4,-50.1,58.9,-38.8Z"
+                transform="translate(100 100)" />
+        </svg>
+    </div>
+</template>
+
+<style>
```

Comment:
Why inline styles?

Reply:
Yes, it was a mistake, i´ll move all inline css to css file